standards/SPIFFE_Workload_API.md

```diff
@@ -227,6 +218,8 @@ It is often the case that a workload doesn’t know what identity it should assu
 
 In order to support the widest variety of use cases, the X.509-SVID Profile supports the issuance of multiple identities, while also defining a default identity. It is expected that workloads which are aware of multiple identities can handle decision making on their own. Workloads which don’t understand how to leverage multiple identities may use the default identity. The default identity is the first in the `svids` list returned in the `X509SVIDResponse` message. Protocol buffers ensure that the order of the list is preserved.
 
+Workloads that understand how to use multiple identities may leverage the optional `hint` field, which can be used to disambiguate identities and inform the workload of which identity should be used for what purpose. For example, `internal` and `external` to denote an SVID for internal or external use, respectively. SPIFFE Workload API implementations SHOULD NOT support values of more than 1024 bytes in length. The exact value of the `hint` field is an operator choice and is otherwise unconstrained by this specification.
```

Comment:
Is there any coordination required between workload and control plane to determine whether a workload "understands" how to use multiple identities?
Should we describe here (or somewhere more appropriate) what happens if a workload doesn't know how to handle a particular `hint` value? Should a workload raise a failure to the control plane? When making a `X509SVIDRequest`, should a workload be able to describe the set of supported `hint` values?

Reply:
How do we manage the potential misalignment between hint values provided by the control plane and hints that the workload wants or needs?

```suggestion
Workloads that understand how to use multiple identities may leverage the optional `hint` field, which can be used to disambiguate identities and inform the workload of which identity should be used for what purpose. For example, `internal` and `external` to denote an SVID for internal or external use, respectively. SPIFFE Workload API implementations SHOULD NOT support values of more than 1024 bytes in length. The exact value of the `hint` field is an operator choice and is otherwise unconstrained by this specification.

It is the workload's decision on how to handle the absence of an expected hint, or the presence of an unexpected one.
```

Optionally, we can add `supported_hints` field on the request message so control plane can log or otherwise misalignments

Reply:
```suggestion
Workloads that understand how to use multiple identities may leverage the optional `hint` field, which can be used to disambiguate identities and inform the workload of which identity should be used for what purpose. For example, `internal` and `external` to denote an SVID for internal or external use, respectively. SPIFFE Workload API implementations SHOULD NOT support values of more than 1024 bytes in length. The exact value of the `hint` field is an operator choice and is otherwise unconstrained by this specification.

It is the workload's responsibility to handle the absence of an expected hint, or the presence of an unexpected one (e.g. fail, warn, etc).
```